src/Layout/Presentation/RemainderQuotients/RemainderQuotients.tsx

```diff
@@ -65,10 +65,11 @@ export class RemainderQuotients extends React.Component<RemainderQuotientsProps>
                 Cell: (row) => {
                     if (row.value !== undefined) {
                         let quotient = row.value.quotient;
-                        if (this.props.year < 2005 && isQuotientAlgorithm(this.props.algorithm)) {
-                            quotient = quotient / 10000;
-                        } else if (this.props.year >= 2005 && isQuotientAlgorithm(this.props.algorithm)) {
+                        const useAdjustedQuotient = this.props.year >= 2005;
+                        if (useAdjustedQuotient && isQuotientAlgorithm(this.props.algorithm)) {
                             quotient = quotient * 10000;
+                        } else if (isQuotientAlgorithm(this.props.algorithm)) {
+                            quotient = quotient / 10000;
```

Comment:
Remind me what the significance of `10000` is here?

Reply:
It is related to whether or not adjusted quotients were used. But it seems a bit strange, this division/multiplication makes quotients for some years in the range of 1000-10 000, and others 0.9-0.01.

Reply:
Isn't there a name for such a constant?

Reply:
A scalar perhaps?

Reply:
I meant what is its role in the formula? Anyway, it will likely be picked up when we set up goals for refactoring at some point, and doesn't have to be dealt with for this release.